sciolyid/functions.py

```diff
@@ -483,32 +471,51 @@ async def get_all_users(bot):
     logger.info("User cache finished")
 
 
-def spellcheck_list(word_to_check, correct_list, abs_cutoff=None):
-    for correct_word in correct_list:
-        if abs_cutoff is None:
-            relative_cutoff = math.floor(len(correct_word) / 3)
-        if spellcheck(word_to_check, correct_word, relative_cutoff) is True:
-            return True
-    return False
+def get_all_categories():
+    """ return all categories, including aliases """
+    # TODO make this a global?
+    return set(
+        list(groups.keys())
+        + [
+            item
+            for group in groups
+            for item in config.options["category_aliases"][group]
+        ]
+    )
+
+
+def dealias_group(group):
+    """ resolve group to a real category by expanding aliases"""
+    return next(
+        key
+        for key, value in config.options["category_aliases"].items()
+        if group in value
+    )
+
+
+def spellcheck_list(word_to_check, correct_list, id_list):
+    return any(
+        spellcheck(word_to_check, correct_word, id_list)
+        for correct_word in correct_list
+    )
 
 
-def spellcheck(worda, wordb, cutoff=3):
+def spellcheck(worda: str, wordb: str, id_list):
```

Comment:
have you tested this new spellcheck on how it performs on different inputs compared to the old spellcheck? the ratio in `get_close_matches` may need tweaking.
also, is there a reason you want `get_close_matches` instead of the previous method?